about.html

```diff
@@ -0,0 +1,131 @@
+    <ul class="nav-menu">
+      <li class="nav-item">
+        <a href="./index.html" class="nav-link">Homepage</a>
+      </li>
+      <li class="nav-item">
+        <a href="#featured" class="nav-link">Featured Artists</a>
+      </li>
+      <li class="nav-item">
+        <a href="#" class="nav-link">Tickets</a>
+      </li>
+      <li class="nav-item">
+        <a href="#" class="nav-link">Info</a>
+      </li>
+    </ul>
+    <div class="hamburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div>
+      <ul class="menu-desktop">
+        <li><a href="./index.html">Homepage</a></li>
+        <li>Lineup</li>
+        <li>Tickets</li>
+        <li>Info</li>
+      </ul>
+  </nav>
+    <div id="about" class="about bg-orange-light-2 pb-2 pt-4">
+      <h2 class="h4-about pt-1 pb-2">green Sziget
+      </h2>
+      <h1 class="h1-about pb-2">SZIGET Festival 2022</h1>
+      <div class="card p-1 m-1 col-10-md col-8-lg col-6-xl">
+        <p class="card-body">One of the Love Revolution’s main message is about environmental consciousness and sustainable development. 
+          Last year, we put a lot of effort into fulfilling our goals and becoming greener and plan on continuing with this tradition this year. 
+          The Love Revolution is our way of celebrating, that together, we can achieve sustainable development and environmental awareness in our daily lives.
+        </p>
+      </div>
+      <div class="contact-info p-1 mt-1 ml-2 mr-2 col-7-md col-5-lg col-4-xl"> 
+        <p>we are always here for you on the following email address: <br>
+          <a href="mailto: hello@szigetfestival.com">hello@szigetfestival.com</a> <br> 
+          If you have questions regarding ticketing purpose, please contact us here: <br> 
+          <a href="mailto: ticket@szigetfestival.com">ticket@szigetfestival.com</a>
+        </p>
+      </div>
+    </div>
+    <div class="sziget-logo p-2 mt-2 mb-4">
+      <img src="./images/sziget-logo.jpeg" alt="sziget logo">
+    </div>
+    <div class="past-fests bg-yellow-light-9">
+      <h2 class="text-black p-2">Check out the SZIGET of past years</h2>
+      <div class="separator"></div>
+      <p class="text-black pl-2 pr-2 mb-2">Take a look at the previous Sziget festivals and get lost in their atmosphere!</p>
+      <a class="past-h" href="https://www.facebook.com/media/set/?set=a.10158890340318975&type=3">
+        <p class="pf-2019">
+          2019
+        </p>
+      <a class="past-h" href="https://www.gettyimages.com.mx/fotos/sziget-festival-2018">
+        <p class="pf-2018">
+          2018
+        </p>
+      </a>
+      <a class="past-h" href="https://www.facebook.com/media/set/?set=a.10156664621338975&type=3">
+        <p class="pf-2017">
+          2017
+        </p>
+      </a>
+      <a class="past-h" href="https://www.facebook.com/media/set/?set=a.10155265083998975&type=3">
+        <p class="pf-2016">
+          2016
+        </p>
```

Comment:
- [ ] Please arrange these images horizontally in the desktop design. The images should only be arranged vertically for the mobile version.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/190466680-a9629fa3-72a0-4c2a-892f-adcd8d158b8c.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/190466776-aff57e05-0309-4554-a448-e12ebf4eb114.png)

Reply:
Thank you, done!